lib/wallaby/browser.ex

```diff
@@ -214,6 +214,10 @@ defmodule Wallaby.Browser do
     path = path_for_screenshot(name)
     File.write! path, image_data
 
+    if opts[:log_screenshot] do
+      IO.puts "Screenshot taken, find it at file:///#{path}"
+    end
```

Comment:
What would you think about changing the option name from `:log_screenshot` to `:log`? That way a call with multiple options would look like:

```elixir
take_screenshot(parent, name: 'my_screenshot.jpg', log: true)
```
Also, please update the docs and the typespec for this function. The `take_screenshot_opt` type should be
```elixir
@type take_screenshot_opt :: {:name, String.t} | {:log, boolean}
```